cmd/kube-spawn/init.go

```diff
@@ -94,5 +95,5 @@ func doInit() {
 		}
 	}
 
-	log.Println("Note: For kubectl to work, please set $KUBECONFIG to $HOME/go/src/github.com/kinvolk/kube-spawn/.kube-spawn/default/kubeconfig.")
+	log.Printf("Note: For kubectl to work, please set $KUBECONFIG to %s\n", utils.GetValidKubeConfig())
```

Comment:
Just a matter of taste, I just think that `Printf("This is %s\n", str)` look better than `Println("This is " + str)`. Doesn't matter actually. Will change it to Println.